website/docs/components/inline-input/inline-input-a11y.md

```diff
@@ -15,22 +15,19 @@ Table: Keyboard support
 | `Tab`         | Moves focus to the next focusable element.     |
 | `Shift + Tab` | Moves focus to the previous focusable element. |
 | `Enter`       | Submits the entered value.                     |
+| `Backspace`   | Erases character or symbol before the cursor.  |
```

Comment:
why is it important to mention this here? Seems redundant, no?

Reply:
It was Glafira's recommendation, but now I'm thinking that yes, looks that we have no need in this here